File: H/Haskell/fibonacci.hs
fibonacci 0 = 1
fibonacci 1 = 1
fibonacci n = fibonacci(n-1) + fibonacci(n-2)

fibonacciN 0 = return()
fibonacciN n = do
    putStr(show(fibonacci n) ++ "\n")
    fibonacciN (n-1)

main = fibonacciN 15

File: src/J/Java/Fibonacci.java
public class Fibonacci {
    public static void main(String[] args) {
        int a = 0, b = 1;
        for (int i = 0; i < 10; i++) {
            System.out.println(a);
            b += a;
            a = b - a;
        }
    }
}